src/views/ConfigurationVideoView.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <BaseConfigurationView>
+  <BaseConfigurationView close-icon>
```

Comment:
I think this and the other commits that add close icons to the menus should be merged with commit `cd82e905` right?

Actually, shouldn't we make the close prop true by default? In this case we don't need the prop explicitly in every configuration view. Seems like having it should be the default, and if we have it as a default=false prop, we will probably forget to add it to future pages.